docs/bugs_and_glitches.md

```diff
@@ -1560,6 +1561,25 @@ The `[hInMenu]` value determines this button behavior. However, the battle moves
 ```
 
 
+### AI Pokémon sometimes do not carry over stat debuffs when PAR/BRN
+
+Sometimes during battle, the AI doesn't get inflicted with the stat debuffs associated with PAR/BRN, especially when switched out and back in for any reason.
```

Comment:
Sounds like we don't understand the circumstances of this bug well enough to accurately describe what it is, let alone how to fix it. How was it observed? Is there some old video of it happening? Has it appeared in GSC bug lists for years and is only now being noticed by pret?

Reply:
I'd recommend asking the person who found this bug in the first place, @Iandesherr